prime-router/src/main/kotlin/azure/BatchFunction.kt

```diff
@@ -132,8 +132,7 @@ class BatchFunction(private val workflowEngine: WorkflowEngine = WorkflowEngine(
             logger.trace("BatchFunction succeeded for message: $message")
         } catch (e: Exception) {
             logger.fatal(
-                "BatchFunction Exception (msg=$message, backstopTime=$backstopTime) : ",
-                e
+                "BatchFunction Exception (msg=$message, backstopTime=$backstopTime) : " + e.stackTraceToString()
```

Comment:
This was a minor unrelated bug I noticed when attempting to track down an error:  the BatchFunction wasn't properly logging stack traces on failures.

Reply:
What was the issue you saw?  The way it was is the correct way to use the logger, so this change may remove some details.  See https://logging.apache.org/log4j/2.x/log4j-api/apidocs/org/apache/logging/log4j/Logger.html#fatal-java.lang.CharSequence-java.lang.Throwable-

Reply:
The issue is that there was no stack trace in the logs.

Reply:
@carlosfelix2 can you resolve your 'request changes'?    The correct way to use the logger is correct, but it wasn't logging the exception in a way that Azure would let us see the logging, so I think the change is a good best effort.   Surprisingly, this is the only logger.fatal in our code right now.

Reply:
I'll approve, but I don't agree with this.  All our other logging follows log4j proper usage, so I think there is a different problem here.